### DISCUSSION TITLE
Relationship between GlobalPlatform and GlobalPlatformPro

### DISCUSSION THREAD

**Discussion by ken4ward** [martinpaljak/GlobalPlatformPro#323] [Q&A] [answered]
I recently stumbled upon many channels with the same identity and in the same field as I try to get started in creating Applets for JavaCards. I'll like to know the Relationship between GlobalPlatform and GlobalPlatformPro. Thanks.

**Comment by martinpaljak** [comment, marked as answer]
global platform is a group publishing standards, globalplatformpro is a software implementing (parts of) those standards.

**Comment by ken4ward** [comment]
Thanks for the thorough explanation. I also read JCOP 5.2 is readily available. Are there JavaCards and Card readers that readily support it? If so, could you help with links to these? Would it be backward compatible with bank ATM and POS already in circulation?
